test/integration/targets/rabbitmq_publish/tasks/ubuntu.yml

```diff
@@ -0,0 +1,180 @@
+- name: Install pika
+  pip:
+    name: pika
+    state: latest
```

Comment:
If there is a minimum version required, specify that. Otherwise just use `present`.

Reply:
Thanks.  Updated.